packages/medusa/src/api/routes/admin/swaps/index.ts

```diff
@@ -24,12 +24,14 @@ export default (app) => {
 export const defaultAdminSwapRelations = [
   "order",
   "additional_items",
+  "additional_items.adjustments",
   "return_order",
   "fulfillments",
   "payment",
   "shipping_address",
   "shipping_methods",
   "cart",
+  "cart.items.adjustments",
```

Comment:
I don't think we can get `cart.items.adjustments` without first getting `cart.items`
```suggestion
  "cart.items",
  "cart.items.adjustments",
```